README.md

```diff
@@ -32,12 +32,59 @@ Build Requirements
 
 Sccache is a [Rust](https://www.rust-lang.org/) program. Building it requires `cargo` (and thus `rustc`). sccache currently requires **Rust 1.21**.
 
-We recommend you install Rust via [Rustup](https://rustup.rs/). The generated binaries can be built so that they are very portable, see [scripts/build-release.sh](scripts/build-release.sh). By default `sccache` supports a local disk cache. To build `sccache` with support for `S3` and/or `Redis` cache backends, add `--features=all` or select a specific feature by passing `s3`, `gcs`, and/or `redis`. Refer the [Cargo Documentation](http://doc.crates.io/manifest.html#the-features-section) for details.
+We recommend you install Rust via [Rustup](https://rustup.rs/). The generated binaries can be built so that they are very [portable](#building-portable-binaries)). By default `sccache` supports a local disk cache. To build `sccache` with support for `S3` and/or `Redis` cache backends, add `--features=all` or select a specific feature by passing `s3`, `gcs`, and/or `redis`. Refer the [Cargo Documentation](http://doc.crates.io/manifest.html#the-features-section) for details.
 
-## Build
+Build
+-----
 
 > $ cargo build [--features=all|redis|s3|gcs] [--release]
 
+### Building portable binaries
+
+When building with the `gcs` feature, `sccache` will depend on OpenSSL, which can be an annoyance if you want to distribute portable binaries. It is possible to statically link against OpenSSL using the steps below before building with `cargo`.
+
+#### Linux
+
+You will need to download and build OpenSSL with `-fPIC` in order to statically link against it.
+
+```
+./config -fPIC --prefix=/usr/local --openssldir=/usr/local/ssl
+make
+make install
+export OPENSSL_LIB_DIR=/usr/local/lib
+export OPENSSL_INCLUDE_DIR=/usr/local/include
+export OPENSSL_STATIC=yes
+```
+
+Build with `cargo` and use `ldd` to check that the resulting binary does not depend on OpenSSL anymore.
+
+#### macOS
+
+Just setting the below environment variable will enable static linking.
+
+```
+export OPENSSL_STATIC=yes
+```
+
+Build with `cargo` and use `otool -L` to check that the resulting binary does not depend on OpenSSL anymore.
```

Comment:
We should probably also mention that `MACOSX_DEPLOYMENT_TARGET` can be set to ensure a minimum version of macOS on which the resulting binary will run.

Reply:
Yes, that makes sense. I see the now removed `scripts/build-release.sh` used to set that.